### PR TITLE
Creation of the Addarr template

### LINKED ISSUE #371: CA Template Request -Addarr
### NOTE
 **If you know how to make unraid container templates PR's are welcome!**
Check out our guide: [Writing a template compatible for Unraid - Selfhosters.net](https://selfhosters.net/docker/templating/templating/)

***

**Please add the dockerhub repo link and github/gitlab link below**
https://github.com/Waterboy1602/Addarr

#### Post any additional information about the project that you know is important 
TG bot for adding downloads to Sonarr/Radarr

### PR DESCRIPTION
**Pull request by bravohotel91**
In regards to CA Template Request -Addarr #371, created an Addarr template for use.

Closes #371

### CODE REVIEW

File: templates/addarr.xml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  <Support>https://github.com/Homelabbers/unraid-templates/issues/new/choose</Support>
+  <Project>https://github.com/Waterboy1602/Addarr</Project>
+  <Overview>TelegramBot for Sonarr and Radarr&#xD;
+&#xD;
+This is a Telegram Bot made to add series to Sonarr or movies to Radarr with a couple of commands. You can also communicate with your Transmission/Sabnzbd service to change its download speed.&#xD;
+&#xD;
+You will need to create a config.yaml file and place in the addarr appdata folder. To create this file, you will need to follow the guide on the Addarr wiki to get authenticated to Telegram.&#xD;
+&#xD;
+</Overview>
+  <Config Name="Config" Target="/app/config.yaml" Default="/mnt/user/appdata/addarr/config.yaml" Mode="rw" Description="" Type="Path" Display="always" Required="false" Mask="false">/mnt/user/appdata/addarr/config.yaml</Config>
```
Comment (nwithan8) [inline templates/addarr.xml:18]: Mode should be `ro`(read-only), per Docker Compose instructions: https://github.com/Waterboy1602/Addarr/wiki/Installation-on-Docker#run-using-docker-compose

File: templates/addarr.xml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  <Config Name="Chat ID" Target="/app/chatid.txt" Default="/mnt/user/appdata/addarr/chatid.txt" Mode="rw" Description="" Type="Path" Display="always" Required="false" Mask="false">/mnt/user/appdata/addarr/chatid.txt</Config>
+  <Config Name="Admin ID" Target="/app/admin.txt" Default="/mnt/user/appdata/addarr/admin.txt" Mode="rw" Description="" Type="Path" Display="always" Required="false" Mask="false">/mnt/user/appdata/addarr/admin.txt</Config>
```
Comment (nwithan8) [inline templates/addarr.xml:20]: Mode should be `ro`

**Review by nwithan8 — CHANGES_REQUESTED** [review]
Looks pretty good, a few minor changes.

File: templates/addarr.xml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  <Project>https://github.com/Waterboy1602/Addarr</Project>
+  <Overview>TelegramBot for Sonarr and Radarr&#xD;
+&#xD;
+This is a Telegram Bot made to add series to Sonarr or movies to Radarr with a couple of commands. You can also communicate with your Transmission/Sabnzbd service to change its download speed.&#xD;
+&#xD;
+You will need to create a config.yaml file and place in the addarr appdata folder. To create this file, you will need to follow the guide on the Addarr wiki to get authenticated to Telegram.&#xD;
+&#xD;
+</Overview>
+  <Config Name="Config" Target="/app/config.yaml" Default="/mnt/user/appdata/addarr/config.yaml" Mode="ro" Description="" Type="Path" Display="always" Required="false" Mask="false">/mnt/user/appdata/addarr/config.yaml</Config>
+  <Config Name="Chat ID" Target="/app/chatid.txt" Default="/mnt/user/appdata/addarr/chatid.txt" Mode="ro" Description="" Type="Path" Display="always" Required="false" Mask="false">/mnt/user/appdata/addarr/chatid.txt</Config>
```
Comment (nwithan8) [inline templates/addarr.xml:19]:
```suggestion
  <Config Name="Chat ID" Target="/app/chatid.txt" Default="/mnt/user/appdata/addarr/chatid.txt" Mode="rw" Description="" Type="Path" Display="always" Required="false" Mask="false">/mnt/user/appdata/addarr/chatid.txt</Config>
```

Should remain RW

**Review by nwithan8 — CHANGES_REQUESTED** [review]

**Review by nwithan8 — CHANGES_REQUESTED** [review]
Please open a separate PR for the warp template and remove it from this PR.

**Comment by bravohotel91** [comment]
sorry about that. getting used to pull requests still

**Review by nwithan8 — APPROVED** [review]

**Pull request merged**